src/numeric/processes.jl

```diff
@@ -55,22 +83,31 @@ function discretize(mvn::MvNormal)
     DiscretizedIIDProcess(x, w)
 end
 
-
-# discrete markov processes
-
-type DiscreteMarkovProcess <: DiscreteProcess
-    transitions::Matrix{Float64}
-    values::Matrix{Float64}
+function rand(mvn::MvNormal, args...)
```

Comment:
I'm not sure about this wrapper. It looks a bit weird, since it doesn't have the basic attributes you would expect (sigma, mu) and still doesn't work for semi-definite matrices.
Let's open issues and revisit later.